Exercise_B/index.css

```diff
@@ -2,3 +2,22 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.Blue{
+    background-color: blue;
+    width:100px;
+    height: 100px;
+    float: left;
+}
+.Red{
+    background-color: red;
+    width:100px;
+    height: 100px;
+    margin: auto;
+    clear:left;
+}
+.Green{
+    background-color:green;
+    width:100px;
+    height: 100px;
+    float: right;
+}
```

Comment:
Great use of floats and clear.

I would extract the width and height into a separate class such that you do not repeat the properties over and over again for each block.

Use an ID for the colour and float.
Make a new class that is applied to every html element for the height and width.